src/plugins/workspace/public/components/workspace_form/workspace_permission_setting_input.test.tsx

```diff
@@ -111,4 +112,21 @@ describe('WorkspacePermissionSettingInput', () => {
     fireEvent.click(renderResult.getByLabelText('Delete permission setting'));
     expect(onDeleteMock).toHaveBeenCalledWith(0);
   });
+
+  it('should call onTypeChange with types after types changed', () => {
+    const { renderResult, onTypeChangeMock } = setup({});
+    Object.defineProperty(HTMLElement.prototype, 'offsetHeight', {
```

Comment:
Should reset offset value after test case completed.

Reply:
Shall we use beforeEach and afterEach?

Reply:
I think we don't need put them in before / after if there only one test case use it.